components/ui/tabcounter/src/main/res/layout/mozac_ui_tabcounter_layout.xml

```diff
@@ -7,8 +7,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     tools:background="@drawable/mozac_ui_tabcounter_round_rectangle_ripple"
     tools:layout_height="40dp"
-    tools:layout_width="40dp"
-    tools:parentTag="org.mozilla.components.ui.tabcounter.TabCounter">
```

Comment:
What does this `parentTag` do? Is it needed in this layout?

Reply:
@eliserichards This is just used for the design view in Android Studio. Since this is a merge layout it can't otherwise know what this will be merged into. However, this was wrong here anyway because it should be `mozilla.components.ui.tabcounter.TabCounter` and not `org.mozilla.components.ui.tabcounter.TabCounter` which doesn't exist. In this case, I think it's fine to remove.